### PR TITLE
FocusWriter: add recipe

### PR DESCRIPTION
**Pull request by khallebal**

### CODE REVIEW

File: app-editors/focuswriter/focuswriter-1.6.15.recipe [hunk 1]
```diff
@@ -0,0 +1,87 @@
+SUMMARY="A simple, distraction-free word processor"
+DESCRIPTION="FocusWriter is a simple, distraction-free word processor. \
+It utilizes a hide-away interface that you access by moving your mouse to \
+the edges of the screen, allowing the program to have a familiar look and \
+feel while still getting out of your way \
+So that you can immerse yourself inyour work."
```
Comment (scottmc) [inline app-editors/focuswriter/focuswriter-1.6.15.recipe]: in your

File: app-editors/focuswriter/focuswriter-1.6.15.recipe [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+HOMEPAGE="http://www.fbreader.org"
+COPYRIGHT="Graeme Gott"
```
Comment (scottmc) [inline app-editors/focuswriter/focuswriter-1.6.15.recipe]: 2008-2018

**Comment by khallebal** [comment]
Good catch, it's now fixed.

File: app-editors/focuswriter/focuswriter-1.6.15.recipe [hunk 3]
```diff
@@ -0,0 +1,87 @@
+SUMMARY="A simple, distraction-free word processor"
+DESCRIPTION="FocusWriter is a simple, distraction-free word processor. \
+It utilizes a hide-away interface that you access by moving your mouse to \
+the edges of the screen, allowing the program to have a familiar look and \
+feel while still getting out of your way \
+So that you can immerse yourself in your work."
+HOMEPAGE="https://gottcode.org/"
+COPYRIGHT="Graeme Gott"
```
Comment (scottmc) [inline app-editors/focuswriter/focuswriter-1.6.15.recipe]: COPYRIGHT="2008-2018 Graeme Gott"

**Comment by scottmc** [comment]
try sending the patch upstream as a pull request

**Comment by korli** [comment]
Patch pushed as PR upstream: https://github.com/gottcode/focuswriter/pull/197

**Pull request merged**